snapcraft.yaml

```diff
@@ -0,0 +1,24 @@
+name: fn
+version: '1.0'
+summary: The container native, cloud agnostic serverless platform  
+description: |
+  The container native, cloud agnostic serverless platform.
+grade: stable
+confinement: strict
+base: core18
+parts:
+  fn:
+    plugin: go
+    source: https://github.com/fnproject/cli.git
+    go-importpath: github.com/fnproject/cli
+    build-packages:
+      - build-essential
+apps:
+  fn:
+    command: bin/cli
+    daemon: simple
+    restart-condition: on-abnormal
+    plugs:
+      - home
+      - network
+      - network-bind
```

Comment:
thank you for adding this, this is great. would you mind opening the PR against https://github.com/fnproject/cli as it seems like this is to install the cli? a future may exist where we would have a snap for the server as well and this would be a little conflicting.

Reply:
Will do. Cheers.